sips/sip-x/sip-x-nft-metadata.md

```diff
@@ -187,16 +187,42 @@ pt-BR.json
 ```
 
 ### Properties
-Common properties are 
+Common properties are described in Appendic C.
 
-| Name | Type | Description |
-|------|------|-------------|
-| `collection` | `string` | collection name the token belongs to. See also Appendix A. |
-| `decimals` | `integer` | number of decimals. See also Appendix A. |
-| `id` | `integer` | identifier for NFTs. See also Appendix A. |
-| `created` | `integer` | creation date of the token in unix timestamp | 
-| `symbol`  | `string` | token symbol |
-| `ip_document_uri` | `string` | link to document about intellectual property (IP) rights |
+### Properties
+Common properties are described in appendix C. Properties of type object are usually described using the following schema:
+```
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Token Metadata Property",
+    "type": "object",
+    "required": ["value"],
+    "properties": {
+        "type": {
+            "type": "string",
+            "description": "type of the property"
+        },
+        "description": {
+            "type": "string",
+            "description": "description of the property"
+        },
+        "value": {
+            "type": any,
```

Comment:
does `type: any` have complications if we wanted to bring this metadata into a contract?
esp given how strongly typed clarity is? you can't even cast a num to an int there.

Reply:
Is `any` a valid type? https://stackoverflow.com/a/16826141/1141891

Reply:
Has been corrected